Makefile

```diff
@@ -271,7 +271,7 @@ generate-metrics-docs:
 .PHONY: generate-go
 generate-go:
 	@echo "Generating mocks..."
-	@go generate ./...
+	@go run github.com/vektra/mockery/v2@v2.32.4
```

Comment:
I am getting `../../go/pkg/mod/github.com/vektra/mockery/v2@v2.32.3/cmd/mockery.go:13:2: reading github.com/chigopher/pathlib/go.mod at revision v1.0.0: unknown revision v1.0.0` without version prefix somehow...

Reply:
🤔 what version of Go do you have installed locally?

Reply:
`go version go1.20.7 linux/amd64`

Reply:
Interesting, looks like v1.0.0 of github.com/chigopher/pathlib, which mockery v2.32.3 depends on, was subsequently retracted (https://github.com/chigopher/pathlib/blob/master/go.mod#L20-L23), but mockery v2.32.4 reverts to an older version: https://github.com/vektra/mockery/blob/v2.32.4/go.mod#L6

Looks like I have v1.0.0 of github.com/chigopher/pathlib installed locally, so that's probably why I don't get the same error.

I think if we upgrade the version of mockery and ensure we're using v0.15.0 for github.com/chigopher/pathlib, your error will go away.

Reply:
It's merged now

Reply:
It's working now, thanks👍